routes/main.js

```diff
@@ -13,6 +13,22 @@ exports.deploy = {
     const defaultService = request.server.settings.app.defaults || {};
     const hostData = request.server.settings.app.hosts;
 
+    server.log(['docker-autostart', 'notice'], { host: request.headers.host, userAgent: request.headers['user-agent'] });
+
+    if (request.server.settings.app.userAgentSkip) {
+      const ua = request.headers['user-agent'];
+      let skip = false;
+      request.server.settings.app.userAgentSkip.forEach(skipAgent => {
```

Comment:
where is this set?

Reply:
I have it set up as an array - so it would be set into the rapptor config file.

Reply:
can we set a default?